config/deploy/server_settings.rb

```diff
@@ -1,31 +1,9 @@
 # Note: this file is required in config/deploy/<environment>.rb
 
 puts "ENV['APP_ENV']  = #{ENV['APP_ENV']}"
-puts "ENV['SDR_APP']  = #{ENV['SDR_APP']}"
-puts "ENV['SDR_USER'] = #{ENV['SDR_USER']}"
-puts "ENV['SDR_HOST'] = #{ENV['SDR_HOST']}"
-puts "ENV['SDR_URL']  = #{ENV['SDR_URL']}"
 puts
 
-set :default_env, {
-    'APP_ENV'  => ENV['APP_ENV'],
-    'SDR_APP'  => ENV['SDR_APP'],
-    'SDR_USER' => ENV['SDR_USER'],
-    'SDR_HOST' => ENV['SDR_HOST'],
-    'SDR_URL'  => ENV['SDR_URL'],
-}
-
-server ENV['SDR_HOST'], user: ENV['SDR_USER'], roles: %w{app}
-Capistrano::OneTimeKey.generate_one_time_key!
-ssh_opts = fetch(:ssh_options)
-ssh_opts[:forward_agent] = true
-ssh_opts[:verbose] = false
-# The :ssh_options are set by
-# capistrano-one_time_key/blob/master/lib/capistrano/tasks/one_time_key.rake
```

Comment:
`deploy/server_settings.rb` is not a pattern common to the rest of the DLSS projects, so I'm  working toward getting rid of it. The pattern of `generating_one_time_key!` occurs in all the other projects in the stage files (see https://github.com/sul-dlss/sul-embed/blob/master/config/deploy/dev.rb#L6) so I'd like to move in the direction of consistency.